.github/workflows/ci.yaml

```diff
@@ -0,0 +1,13 @@
+name: Run Unit Tests
+on:   
+  pull_request:
+    branches:
+      - '**'
```

Comment:
I think we can remove this line? 

Its currently not running on PR this reason (I believe)

Reply:
it's not running because there is a security feature. as to prevent attackers from creating workflows that have some costs to them (gpu or other cpu intensive requests/target machines)